Source/WebCore/platform/graphics/MediaPlayerPrivate.h

```diff
@@ -336,6 +336,8 @@ class MediaPlayerPrivateInterface {
     virtual void setResourceOwner(const ProcessIdentity&) { }
 
     virtual String errorMessage() const { return { }; }
+
+    virtual void renderVideoWillBeDestroyed() { }
```

Comment:
I don't understand why `acceleratedRenderingStateChanged()` is insufficient here. Destroying the renderer should result in that method being called, and `renderingCanBeAccelerated()` should start returning `false`, and `MediaPlayerPrivateMediaStreamAVFObjC::destroyLayers()` should be called.

Reply:
Ah, but that would affect all MediaPlayerPrivateInterface subclasses, and you're trying to do the less risky change here.

Reply:
Right, I am only changing MediaPlayerPrivateMediaStreamAVFObjC in this patch.